pvactools/lib/run_argument_parser.py

```diff
@@ -245,6 +245,12 @@ def __init__(self):
             default=False,
             action='store_true'
         )
+        self.parser.add_argument(
+            "--tumor-purity",
+            help="Value between 0 and 1 indicating the fraction of tumor cells in the tumor sample. Information is used during aggregate report creation for a simple estimation whether variants are subclonal or clonal based on VAF.",
```

Comment:
Suggested change (here and below): 
help="Value between 0 and 1 indicating the fraction of tumor cells in the tumor sample. Information is used during aggregate report creation for a simple estimation of whether variants are subclonal or clonal based on VAF. If not provided, purity is estimated directly from the VAFs"